### DISCUSSION TITLE
Estimated time remaining?

### DISCUSSION THREAD

**Discussion by Plutoaurus** [winder/Universal-G-Code-Sender#2553] [Help]
Hi

Not sure if I'm being blind but is there a estimated time anywhere?

I have only found in the send progress which is 0 until you start.

Want to get a rough time a file would take.

Thanks

**Comment by breiler** [comment]
In the bottom. It shows the progressed time and the estimated time:

![Screenshot from 2024-06-18 06-36-06](https://github.com/winder/Universal-G-Code-Sender/assets/8962024/16d67684-025f-4baf-99fd-16e0e0d45deb)

**Reply by Plutoaurus** [reply]
Do you have to have it running for it to show this?
Trying to move from another sender where it gave a estimated time before starting.

It's only the greyscale relief program I use to create the gcode doesn't offer a time.

Thanks
